app/code/Magento/Fedex/Model/Source/Generic.php

```diff
@@ -1,11 +1,18 @@
 <?php
+
 /**
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
 namespace Magento\Fedex\Model\Source;
 
-class Generic implements \Magento\Framework\Option\ArrayInterface
+/**
+ * Fedex generic source implementation
+ *
+ * @author Magento Core Team <core@magentocommerce.com>
```

Comment:
`@author` ,`@category`, `@package`, and `@subpackage` MUST NOT be used. Documentation is organized with the use of namespaces.

https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#documentation-space